.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug Report
+about: Create a bug report.
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**What happened?**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior: 
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
```

Comment:
Should this be combined into the first question. I'm not sure if this is redundant.